src/SpendPermissionManager.sol

```diff
@@ -81,6 +82,9 @@ contract SpendPermissionManager is EIP712 {
     ///         (https://eips.ethereum.org/EIPS/eip-6492).
     PublicERC6492Validator public immutable publicERC6492Validator;
 
+    /// @notice MagicSpend singleton (https://github.com/coinbase/magic-spend).
+    address public immutable magicSpend;
```

Comment:
an automated finding in the lightchaser report (the report we never knew about from Spearbit at the beginning of the competition audit) notes that anything constant or immutable should be in CONSTANT_CASE.

Maybe we should make MAGIC_SPEND and PUBLIC_ERC6492_VALIDATOR constant case? Not opinionated, just calling it out.

Reply:
sounds good to me!